src/Entity/User.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\Collection;
+
+class User
+{
+    private int $id;
+
+    private string $password;
+
+    private Collection $roles;
+
+    private string $cnp;
+
+    public string $firstName = '';
+
+    public string $lastName = '';
+
+    private Collection $programs;
+
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    public function setId(int $id): void
+    {
+        $this->id = $id;
+    }
+
+    public function getPassword(): string
+    {
+        return $this->password;
+    }
+
+    public function setPassword(string $password): void
```

Comment:
why don't you return `$this` in the case of this method?

Reply:
Ok. I have used Code->generate setters and getters and then refactor...